lib/src/features/wine_tasting_create_view/bloc/wine_tasting_create_bloc.dart

```diff
@@ -100,6 +110,34 @@ class WineTastingCreateBloc extends Bloc<WineTastingCreateEvent, WineTastingCrea
     _inNotesCategorized.add(notesCategorized);
   }
 
+  /// Custom Sliders Stream
+  // Controller: Page <- App Database.
+  final _getSlidersController = BehaviorSubject<List<CustomSlider>>();
+
+  // Stream: In
+  // Purpose: Update stream that pages subscribe to.
+  StreamSink<List<CustomSlider>> get _inSliders => _getSlidersController.sink;
+
+  // Stream: out.
+  // Purpose: Stream that other pages subscribe to for notes.
+  Stream<List<CustomSlider>> get sliders => _getSlidersController.stream.asBroadcastStream();
+
+  void refreshSlidersStreamAndCharacteristics() async {
+    // Retrieve all the notes from the database.
+    var sliders = await sliderRepository.getAllSliders();
+
+    sliders.asMap().forEach((index, slider) {
+      characteristics[slider.name] = Characteristic(
```

Comment:
Destroy the previous map to open the door to deleting characteristics in the future